### PR TITLE
Grammar

### PR DESCRIPTION
**Pull request by chrisfinazzo**

### CODE REVIEW

File: site/_docs/configuration.md [hunk 1]
```diff
@@ -448,9 +448,8 @@ The `projects/foo_project.md` would have the `layout` set to `foobar` instead of
 
 ## Default Configuration
 
-Jekyll runs with the following configuration options by default. Unless
-alternative settings for these options are explicitly specified in the
-configuration file or on the command-line, Jekyll will run using these options.
+Jekyll runs with the following configuration options by default. Alternative settings for these options can be explicitly specified in the
```
Comment (parkr) [inline site/_docs/configuration.md]: Would you mind sticking to the hard-wrap at 80 characters here?

File: site/_docs/continuous-integration.md [hunk 2]
```diff
@@ -69,8 +69,7 @@ check out `html-proofer`'s README file.
 
 This file is used to configure your Travis builds. Because Jekyll is built
 with Ruby and requires RubyGems to install, we use the Ruby language build
-environment. Below is a sample `.travis.yml` file, and what follows that is
-an explanation of each line.
+environment. Below is a sample `.travis.yml` file, followed by an explanation of each line.
```
Comment (parkr) [inline site/_docs/continuous-integration.md]: Same here with the 80 character hard-wrap :smile:

File: site/_docs/deployment-methods.md [hunk 3]
```diff
@@ -107,8 +107,7 @@ If you want to maintain Jekyll inside your existing Rails app, [Jekyll-Admin](ht
 
 ## Amazon S3
 
-If you want to host your site in Amazon S3, you can do so with
-[s3_website](https://github.com/laurilehmijoki/s3_website) application. It will
+If you want to host your site in Amazon S3, you can do so by using the [s3_website](https://github.com/laurilehmijoki/s3_website) application. It will
```
Comment (parkr) [inline site/_docs/deployment-methods.md]: Does this markdown render properly?

File: site/_docs/configuration.md (hunk 1 above)
Comment (chrisfinazzo) [inline site/_docs/configuration.md]: Oh, yeah, my bad. I have BBEdit set to wrap text at its "Page Guide" which is 72 characters.

File: site/_docs/deployment-methods.md (hunk 3 above)
Comment (chrisfinazzo) [inline site/_docs/deployment-methods.md]: Looking at the entire file again, it should be fine. However, this brings to mind a question. Does the Jekyll site have a policy on how links should be handled? IMHO, inline links are gross, and if you're doing that, it isn't that much of a stretch to just use HTML for everything, thus negating any advantage you get from using Markdown.

Seems easier to read if things are separated (e.g, reference links). Thoughts?

**Comment by chrisfinazzo** [comment]
Yeah, I know this didn't work. I don't use the line wrapping features all that often, usually just the page guide. Somehow this diverged from my branch and it's not coming back together...

merge

**Comment by parkr** [comment]
Try opening a new PR?

**Comment by chrisfinazzo** [comment]
Yeah, that sounds like the best thing to do, I'll rollback to the original fix and make another run at this... 😡 👎

**Comment by parkr** [comment]
Sorry :(

**Comment by chrisfinazzo** [comment]
See #3396.

**Pull request closed without merging**